dev/sg/internal/std/output.go

```diff
@@ -33,8 +33,8 @@ func NewOutput(dst io.Writer, verbose bool) *Output {
 
 	return &Output{
 		Output: output.NewOutput(dst, output.OutputOpts{
-			ForceColor: true,
-			ForceTTY:   true,
+			ForceColor: false,
+			ForceTTY:   false,
 			Verbose:    verbose,
```

Comment:
In that case we can rely on the default values. (With "Force..." options/params I think their presence/absence is often what reading eyes latch onto, not the value, so removing those here and relying on default value helps understanding that no "force*" params are used here, IMH)

```suggestion
			Verbose: verbose,
```

Reply:
Good point, agreed.